plugins/polio/api/lqas_im/lqasim_zoom_in_map.py

```diff
@@ -124,12 +124,12 @@ def list(self, request):
                 continue
             if latest_active_campaign.separate_scopes_per_round:
                 scope = latest_active_campaign.get_districts_for_round_number(round_number)
-
             else:
                 scope = latest_active_campaign.get_all_districts()
             # Visible districts in scope
+            scope_qs = OrgUnit.objects.filter(id__in=[ou.id for ou in scope])
```

Comment:
I don't understand this one: both `get_districts_for_round_number` and `get_all_districts` return a queryset that already filtered the OrgUnits in the scope